process/smartContract/builtInFunctions/esdtMetaData.go

```diff
@@ -0,0 +1,65 @@
+package builtInFunctions
+
+const lengthOfESDTMetadata = 2
+
+const (
+	METADATA_PAUSED = 1
```

Comment:
Please rename to camel case, that's how we use constants everywhere

Reply:
done